mmcv/runner/hooks/logger/wandb.py

```diff
@@ -14,13 +17,28 @@ def __init__(self,
                  reset_flag=False,
                  commit=True,
                  by_epoch=True,
-                 with_step=True):
+                 with_step=True,
+                 config_path=None):
+        
         super(WandbLoggerHook, self).__init__(interval, ignore_last,
                                               reset_flag, by_epoch)
+        
+        """
+        Args:
+            
+            with_step (bool): whether making a log in each step or not.
+                        Default: True.
+            config_path (str, optional): The path of final config of each experiment.
+                        It can be either a path of final config file or a directory of 
+                        config files. The config is uploaded to wandb server if it is
+                        not None. Default: None.
+        """
+        
```

Comment:
This part is the docstring of wandb logger class. I wrote docs for only two arguments because the author of this class is not I. I worried that I can't wrote the exact function.

Reply:
Yet, two arguments are enough here. BTW, the convention of mmcv is that put the docstring on top of `__init__` method so the docstring should be moved to line 12.